plugin/snomed-ct-starterdata-maven-plugin/src/main/java/dev/ikm/maven/SnomedStarterDataMojo.java

```diff
@@ -34,9 +34,9 @@ public class SnomedStarterDataMojo extends AbstractMojo
 {
     @Parameter(property = "origin.namespace", required = true)
     String namespaceString;
-    @Parameter(property = "datastore.path", required = true)
+    @Parameter(property = "datastorePath", required = true)
```

Comment:
Naming scheme for these parameters has gone from '.' separated to camel case for these last two. I would stick with one or the other.